Resources/doc/index.rst

```diff
@@ -0,0 +1,13 @@
+FOSJsRoutingBundle
+==================
+
+This bundle allows to expose Symfony Routes to JavaScript, so you can generate
+relative or absolute URLs in the browser using the same routes as in the backend.
+
+.. toctree::
+    :maxdepth: 1
+
+    installation
+    usage
+    compilling
+    testing
```

Comment:
compiling and testing might be placed in a separate section than installation and usage, as they are about contributing to the bundle, not about using it (or their content might be moved to a CONTRIBUTING.md file rather than the doc)